docs/package.json

```diff
@@ -39,15 +55,12 @@
       "not op_mini all"
     ],
     "development": [
-      "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
+      "last 3 chrome version",
+      "last 3 firefox version",
+      "last 5 safari version"
     ]
   },
-  "devDependencies": {
-    "@docusaurus/types": "^2.0.0-beta.1",
-    "@typescript-eslint/eslint-plugin": "^5.62.0",
-    "@typescript-eslint/eslint-plugin-tslint": "^5.62.0",
-    "@typescript-eslint/parser": "^5.62.0"
+  "engines": {
+    "node": ">=18.0"
```

Comment:
heads up, can't remember if we host this via Netlify or not, but you'll want to make sure its runner/deploy is updated to this Node version if you want to specify it like this

it's quite possible it's running a deprecated v14/16 version in its config

Reply:
Confirmed that we're good here--I checked the latest deploy log and it is using node 18 already from `.nvmrc` and deployed a preview just fine